scripts/set_kokkos_env.sh

```diff
@@ -0,0 +1,12 @@
+# set nvcc as the underlying C++ compiler for Open MPI
+export OMPI_CXX=$TRILINOS_DIR/packages/kokkos/config/nvcc_wrapper
+
+# set nvcc as the underlying C++ compiler for MPICH
+export MPICH_CXX=$TRILINOS_DIR/packages/kokkos/config/nvcc_wrapper
+
+# set the original C++ compiler as the underlying compiler for the nvcc
+# wrapper
+export NVCC_WRAPPER_DEFAULT_COMPILER=$PATH_TO_CXX_COMPILER
+
+# set all cuda calls to blocking launch
+export CUDA_LAUNCH_BLOCKING=1
```

Comment:
DTK does not necessarily need UVM, but I think Tpetra does. As such, the build needs UVM. I also don't think Tpetra has proper fences in place, so UVM would require having CUDA_LAUNCH_BLOCKING.